samples/bot-adaptive-card-actions/nodejs/Readme.md

```diff
@@ -0,0 +1,133 @@
+---
+page_type: sample
+description: Demonstrating the feature where user can send Adaptive Card actions using bot.
```

Comment:
Demonstrating the feature where user can send adaptive card with different actions using bot.

Reply:
fixed